example_rules/example_single_metric_agg.yaml

```diff
@@ -0,0 +1,35 @@
+name: Metricbeat CPU Spike Rule
+type: metric_aggregation
+
+es_host: localhost
+es_port: 9200
+
+index: metricbeat-*
+
+buffer_time:
+  hours: 1
+
+metric_agg_key: system.cpu.user.pct
```

Comment:
This is the same syntax that Elasticsearch uses for querying against nested fields. If we are trying to read from the _source document, then we need to use lookup_es_key, but not for metrics/queries.